x-pack/plugins/audit_trail/server/types.ts

```diff
@@ -13,4 +13,5 @@ export interface AuditEvent {
   scope?: string;
   user?: string;
   space?: string;
+  requestId?: string;
```

Comment:
This is probably not the right name since this identifier may not come from a request (eg. background task).

Reply:
In ECS audit log this will be logged as `trace.id` but I'd say don't worry about updating the schema for now. I am happy to implement this as part of audit log PR as it's just going to create merge conflicts.